geopandas/_vectorized.py

```diff
@@ -156,9 +158,10 @@ def to_shapely(data):
         return data
 
 
+@doc(type="a np.ndarray[geoms]")
 def from_wkb(data):
     """
-    Convert a list or array of WKB objects to a np.ndarray[geoms].
+    Convert a list or array of WKB objects to {type}.
```

Comment:
This docstring is not part of a public API. It is here for people reading the code to understand what it does. I don't see the benefit of moving the type to a decorator, it just impairs the readability of the code.